samples/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6-slim
+VOLUME /tmp
+ADD ./java/greeter/target/greeter-1.0.0.jar /functions/java/greeter-1.0.0.jar
```

Comment:
We should probably restructure our samples so that this will be future-proof (can do that as a separate PR after this one is merged for now). For example, any new samples' code should be included in this. Likewise, all the yamls should be in one place (not necessarily the root "samples" dir).

Reply:
although, samples deploy/svc is separate from the core faas implementation and would belong outside of the config directory